apps/microtvm/zephyr/template_project/src/host_driven/main.c

```diff
@@ -218,11 +303,42 @@ void uart_irq_cb(const struct device* dev, void* user_data) {
   }
 }
 
+#ifdef FVP
```

Comment:
is this code needed any longer?

Reply:
we are using the uart to write some log messages, and it might be a good idea to initialize the fvp uart correctly. but technically we don't use the fvp uart in RX mode, and it won't cause an error if we remove these lines (unless someone accidentally sends data to the corresponding TCP port, which could cause an overflow!)

Reply:
yeah i think if possible let's rely on Zephyr for UART TX

Reply:
done